did-pkh-method-draft.md

```diff
@@ -0,0 +1,256 @@
+the blockchain where it is valid.
+
+### Read (Resolve)
+
+Resolution implements the following interface defined in [DID Core][]:
+```
+resolve(did, resolutionOptions) →
+   « didResolutionMetadata, didDocument, didDocumentMetadata »
+```
+
+Construct the DID Document for *did* as follows:
+- Parse the DID into its network id, *network* and account address, *address*,
+  according to Syntax and Interpretation above.
+- Initialize a DID document, *doc*, as a JSON-LD document.
+- Set the `id` property of *doc* to *did*.
+- Set the `@context` property of *doc* to an array,
+  `["https://www.w3.org/ns/did/v1", context]`, where `context` is the [did:pkh JSON-LD context object](#context).
+- Construct the verification method ID, *vm*, by appending
+  "#blockchainAccountId" to *did*.
+- Construct the [verification method][] object, *vmObj* as follows:
+  - Insert property `id` into *vmObj* with value *vm*.
+  - Look up network id *network* in the did:pkh [Networks][#networks] table, to
+    get verification method type *vmType* and [CAIP-2][] chain id *chainId*. If
+    there are multiple entries in the table for *network*, use one that
+    matches *address*.
+  - Insert property `type` into *vmObj* with value *vmType*.
+  - Insert property `controller` into *vmObj* with value *did*.
+  - Construct string *accountId* by concatenating *address* + "@" + *chainId*.
+  - Insert property `blockchainAccountId` into *vmObj* with value *accountId*.
+- Insert a property into *doc* with key name `verificationMethod` and a value of
+  an array containing only *vmObj*.
+- Insert a property into *doc* with key name `authentication` and a value of an
+  array containing only *vm*.
+- Insert a property into *doc* with key name `assertionMethod` and value of an
+  array containing only *vm*.
+- Construct an empty DID Resolution metadata object, *resMeta*.
+- Construct an empty DID Document metadata object, *docMeta*.
+- Return *resMeta*, *doc*, and *docMeta*.
+
+
+### Update
+
+No updates possible. did:pkh DID Documents are, like [did:key] documents, intended for local-only usage.
+
+### Delete
+
+No deletion possible. did:pkh DID Documents are, like [did:key] documents, intended for local-only usage.
+
+## Security & Privacy Considerations (non-normative)
+
+There are a number of security and privacy considerations that implementers will want to take into consideration when implementing this specification. These are adapted from the analogous considerations proposed by the did:key authors.
+
+### Key Rotation Not Supported
+
+The did:pkh method is a purely generative method, which means that updates are not supported. This can be an issue if a did:pkh is expected to be used over a long period of time. For example, if a did:pkh is ever compromised, it is not possible to rotate the compromised key. For this reason, using a did:pkh for interactions that last weeks to months is strongly discouraged.
```

Comment:
Feels like this note should mention that eth addresses and similar ARE being used over longer periods of time right now.

Reply:
But that goes against the assumptions of DID users and creates a false equivalence to other DID methods, doesn't it?

Reply:
What is the assumptions of DID users? Seems like we should be clear about how people are using on-chain addresses now. Ideally PKH DID is just agnostic about how you use it?

Reply:
Proposal: merge as-is, open a tracking issue, agenda said issue for first meeting, and discuss on the first minuted public call? Seems a very CCG topic :D